### ISSUE TITLE
SPA documentation for SvelteKit

### ISSUE THREAD

**Issue by Tommertom** [sveltejs/kit#7269]
### Describe the problem

I'm frustrated when I want to use the next Kit version in SPA mode as the documentation on SPA is always lagging behind. SPA is not the recommended route, but for some people like me - I have to use SPA given some library.

I believe Kit can potentially get more adoption from developers from frameworks like Angular (and maybe React, Vue) who don't use meta-frameworks - so drilled in SPA working -  but want to use the great DX from Svelte nevertheless. 

Going into Kit, while coming from such SPA thinking is a bit steeper learning curve. So, maybe good to spend a bit of time describing SPA setup and common pitfuls working with SPA in Kit (like sometimes needing to import libraries in onMount).

### Describe the proposed solution

I would like to see page https://github.com/sveltejs/kit/edit/master/documentation/docs/13-page-options.md - or some other page - containing a comprehensive description on how to setup SPA in svelte. Previous documentation did mention it a bit better.

I am not sure if the maintainers also wish to provide some specific guidance for people moving from other frameworks. And if not, a SPA guide may help bloggers/vloggers to write some. 

### Alternatives considered

Posting Q&A's on Reddit, asking questions on Twitter, including tagging Svelte team members.

### Importance

nice to have

**Comment by dummdidumm** [comment]
If you add `export const ssr = false` to your root `+layout.js`, you created an SPA - nothing more to do than that. The "import libraries on mount" thing is only needed when you do SSR, too (so, in case of no SPA). Is what you are missing an explicit mention of the term SPA in the documentation, or what exactly is unclear?

**Comment by Tommertom** [comment]
Hi - I had looked into the docs on ways to make it a SPA and decided to do `const prerender = false`, and ran into an issue which I thought I had to solve with onMount.

Obviously this is not correct - my bad.

Nevertheless, I think if there is a sentence in the paragraph saying "With const ssr = false, you basically making the app run as a SPA", that would have triggered me to do differently, and learn more about kit. In fact, I believe such sentence was there in the previous docs.

And if I may, with `const ssr = false`,  when running the build process, there is still a server package generated. Why is that then? (adapter static). In previous Kit version (401) that did not happen - if I recall correctly (just deleted my only 401 project).

In fact, the previous doc mentioned, if you add `some code related to ssr` to hook.ts and use adapter static, you make it a SPA.

I know SPA is not the preferred way to go, but for some cases usefull (like wrapped in Cordova/Capacitor) or inevitable (when using SSR-incompatible libraries).
